src/providers/videos/videos.actions.js

```diff
@@ -0,0 +1,40 @@
+import VideosActionTypes from './videos.types';
+import { searchVideos } from '../../utils/js/youtube.api';
+
+export const fetchVideosStart = () => ({
+  type: VideosActionTypes.FETCH_VIDEOS_START,
+});
+
+export const fetchVideosSuccess = (videos) => ({
+  type: VideosActionTypes.FETCH_VIDEOS_SUCCESS,
+  payload: videos,
+});
+
+export const fetchVideosFailure = (errorMessage) => ({
+  type: VideosActionTypes.FETCH_VIDEOS_FAILURE,
+  payload: errorMessage,
+});
```

Comment:
I like the way you implemented this, I've seen people using a different approach, but this one is interesting, using the reducer makes things easy to test